client/src/schema/pagination.ts

```diff
@@ -26,23 +36,51 @@ export class Page<T extends XataRecord, R extends XataRecord> implements Paginab
     this.records = records;
   }
 
+  /**
+   * Retrieves the next page of results.
+   * @param size Maximum number of results to be retrieved.
+   * @param offset Number of results to skip when retrieving the results.
```

Comment:
Applies to all methods, should we add the type to the param comment? (Usage outside TS)

Reply:
Good suggestion. Not sure about that. I've just created an issue to decide that https://github.com/xataio/client-ts/issues/77